data/projects.json

```
@@ -188,5 +188,15 @@
         "level": "intermediate",
         "site": "https://github.com/Angelvicks/vision-ai",
         "tags": "fast-api, react, artificial-intelligence"
+    },
+    {
+        "name": "poketalog",
+        "display": "Poketalog",
+        "description": "Catalog Pokemon using pokeapi.co",
+        "author": "shonenxnaifu",
+        "category": "website",
+        "level": "intermediate",
+        "site": "http://poketalog.herokuapp.com/",
```

Hi @shonenxnaifu,
Nice work!!, a question does this site has `https`?
> If yes please make the suggestion
```suggestion
        "site": "https://poketalog.herokuapp.com/",
```

---

ok i've changed the url, thanks for the suggestion